transport-interop/impl/js/v1.3/Dockerfile

```
@@ -0,0 +1,11 @@
+# Here because we want to fetch the node_modules within docker so that it's
+# installed on the same platform the test is run. Otherwise tools like `esbuild` will fail to run
+FROM node:18.17.1
```

We should use LTS here:

```suggestion
FROM node:20.12.1
```